frontend/src/i18n/locales/de/frequent.json

```diff
@@ -0,0 +1,124 @@
+  "Appbar Tools": "Appbar-Tools",
+  "show more": "mehr anzeigen",
+  "Version": "Version",
+  "Git Commit": "Git Commit",
+  "Try Again": "Erneut versuchen",
+  "Back": "Zurück",
+  "Next": "Weiter",
+  "Finish": "Oberfläche",
+  "Offline": "Offline",
+  "No": "Nein",
+  "Yes": "Ja",
+  "Toggle fullscreen": "Vollbild umschalten",
+  "Close": "Schließen",
+  "Find": "finden.",
+  "All namespaces": "Alle Namespaces",
+  "Type": "Typ",
+  "Reason": "Grund",
+  "From": "Von",
+  "Message": "Nachricht",
+  "Age": "Alter",
+  "Skip": "Überspringen",
+  "Read more": "Mehr lesen",
+  "Dismiss": "Ablehnen",
+  "Create / Apply": "Erstellen/Bewerben",
+  "Deleting item {{ itemName }}…": "Element {{ itemName }} löschen…",
+  "Cancelled deletion of {{ itemName }}.": "Löschung von {{ itemName }} abgebrochen.",
+  "Deleted item {{ itemName }}.": "Gelöschter Artikel {{ itemName }}.",
+  "Error deleting item {{ itemName }}.": "Fehler beim Löschen des Elements {{ itemName }}.",
+  "Edit": "Bearbeiten",
+  "Editor": "Herausgeber",
+  "Documentation": "Documentation",
+  "Undo": "Rückgängig machen",
+  "Are you sure?": "Sind Sie sicher?",
+  "Undo Changes": "Änderungen rückgängig machen",
+  "Save & Apply": "Speichern & Anwenden",
+  "Loading resource data": "Laden von Ressourcendaten",
+  "Mount Path": "Mount Path",
+  "from": "von",
+  "I/O": "E/A",
+  "Ready": "Bereit",
+  "Waiting": "Warten",
+  "Running": "Läuft",
+  "Error": "Error",
+  "Restart Count": "Anzahl der Neustarts",
+  "Restarting {{ itemName }}…": "Neustart von {{ itemName }}…",
+  "Cancelled restarting {{ itemName }}.": "Der Neustart von {{ itemName }} wurde abgebrochen.",
+  "Restarted {{ itemName }}.": "Neustart von {{ itemName }}.",
+  "Failed to restart {{ itemName }}.": "Neustart von {{ itemName }} fehlgeschlagen.",
+  "Restart": "Neu starten",
+  "restart": "Neustart",
+  "Are you sure you want to restart {{ name }}?": "Sind Sie sicher, dass Sie {{ Name }} neu starten wollen?",
+  "Scaling {{ itemName }}…": "Skalierung {{ itemName }}…",
+  "Cancelled scaling {{ itemName }}.": "Abgebrochene Skalierung {{ itemName }}.",
+  "Scaled {{ itemName }}.": "Skaliert {{ itemName }}.",
+  "Failed to scale {{ itemName }}.": "Skalierung von {{ itemName }} fehlgeschlagen.",
+  "Scale": "Skalieren",
+  "scale": "Skala",
+  "Scale Replicas": "Maßstabsgetreue Replikate",
+  "Decrement": "Verkleinern",
+  "Increment": "Inkrement",
+  "A large number of replicas may negatively impact the cluster's performance": "Eine große Anzahl von Replikaten kann sich negativ auf die Leistung des Clusters auswirken",
+  "Show filter": "Filter anzeigen",
+  "Search": "Suchen",
+  "Filter": "Filter",
+  "Clear": "Klar",
+  "sort up": "einsortieren",
+  "sort down": "aussortieren",
+  "sort swap": "Sortentausch",
+  "Refresh": "Aktualisieren",
+  "Add": "Hinzufügen",
+  "Data": "Daten",
+  "Group": "Gruppe",
+  "Versions": "Versionen",
+  "Full name": "Vollständiger Name",
+  "Spawn": "Laich",
```

Comment:
is this a verb? would be laichen, but not sure if that would be used in this context

Reply:
Yeah, this is a verb (button action for spawning a cronjob).